packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -440,14 +498,20 @@ export default decorate([
       </div>
     ) : (
       <div>
+        <SearchBar
+          className='mb-1'
+          filters={state.searchBarFilters}
+          onChange={effects.onFilterChange}
+          value={state.filter}
+        />
         <Select
           labelKey='label'
-          onChange={effects.setFilter}
+          onChange={effects.onStatusChange}
           optionRenderer={state.optionRenderer}
           options={state.options}
           required
           simpleValue
-          value={state.filter || state.defaultFilter}
+          value={state.status}
```

Comment:
Doesn't it make more sense to pass `_status` (instead of `status`) here?

Reply:
No because the default value will not be selected.